streams/src/main/java/org/apache/kafka/streams/kstream/ForeachProcessor.java

```diff
@@ -19,7 +19,11 @@
 import org.apache.kafka.streams.processor.api.Processor;
 import org.apache.kafka.streams.processor.api.Record;
 
-// deprecated since 4.0.
+/**
+ * This class is deprecated and should not be used any longer.
+ *
+ * @deprecated since 4.0. Use {@link org.apache.kafka.streams.kstream.internals.ForeachProcessor} instead.
```

Comment:
```
/**
 * @deprecates since 4.0 and should not be used any longer.
 */
```

Reply:
@m1a2st please remove the incorrect comment. The sample code from https://github.com/apache/kafka/pull/17129#discussion_r1752406123 is good enough I think